src/Common/tests/Performance/PerfUtils.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.IO;
+
+namespace System
+{
+    /// <summary>Static helper class for performance tests</summary>
+    public static class PerfUtils
+    {
+        private static Random rand = new Random(1234132);
```

Comment:
Can multiple benchmarks run concurrently?  If so, couldn't interleaving mean that different runs a suite test different scenarios?  Is that good or bad (given that we have a fixed seed here it feels like you want things to be deterministic across runs...)